src/fsharp/fsi/fsi.fs

```diff
@@ -1329,7 +1346,7 @@ type internal FsiInterruptController(fsiOptions : FsiCommandLineOptions,
         ignore fsiConsoleOutput
         ignore CTRL_C
         ignore fsiOptions
-        false
+        exitViaKillThread <- false
```

Comment:
We should work out what this change is for (was it a mistake in the FCS integration?)